### ISSUE TITLE
Support displaying different contracts/target images depending on gameversion

### ISSUE THREAD

**Issue by suanjiansalt** [thepeacockproject/Peacock#404]
**What**
From H2016 to H2 and H3, images of some (Elusive Target) missions have been changed, including:

- The Sensation (changed in H3)
- The Identity Thief (changed in H2)
- The Chef (changed in H3)
- The Food Critic (changed in H3)
- The Chameleon (changed in H2)
- The Warlord (changed in H3)
- The Badboy (changed in H3)
- The Undying Returns (changed in H3?)

**Why**
As official does.

**Comment by RDIL** [comment]
I honestly think this is fine the way it is - the new images look great, even in the older engine versions.

**Comment by suanjiansalt** [comment]
> I honestly think this is fine the way it is - the new images look great, even in the older engine versions.


Yes, the new images show the progress of IO's aesthetics. But in my humble opinion these images are a bit too vibrant and don't quite fit in with the overall stylistic grey and white atmosphere of 2016.

**Comment by AnthonyFuller** [comment]
Lets get this in for v8.

**Comment by AnthonyFuller** [comment]
Lets get this in for v8. Although you can install the Offline Image Pack mod for both of those versions ([H2016](https://www.nexusmods.com/hitman2016/mods/2) or [H2](https://www.nexusmods.com/hitman2/mods/107)) to get the proper engine-specific images.

**Comment by suanjiansalt** [comment]
The question is, in which paths do we save these alternative images.
